scripts/installer/postgres-setup

```diff
@@ -31,13 +32,21 @@ my %CONFIG_DEFAULTS = (
 
 my %CONFIG_PROMPTS = (
     'POSTGRES_SERVER',   'Postgres Server Address',
+    'POSTGRES_SERVICE',  'Postgres Service Address',
     'POSTGRES_PORT',     'Postgres Server Port',
     'POSTGRES_ADMIN_PASSWORD', 'Postgres ADMIN password',
     'POSTGRES_DATABASE', 'Name of the Postgres Database',
     'POSTGRES_USER',     'Name of the Postgres User',
     'POSTGRES_PASSWORD', 'Postgres user password',
 );
 
+$CONFIG_DEFAULTS{'POSTGRES_PORT'} = $ENV{'DB_PORT'};
```

Comment:
Why was this necessary for all the existing values?

Reply:
Most of this code was duplicated a few lines lower, so I deleted duplicate code in this block.